doc/changelog.md

```diff
@@ -40,6 +40,29 @@
   f(2, MyClass(5)) # no re-compilation
   ```
 
+* Catalyst now supports QJIT compatible `catalyst.vmap` of hybrid programs.
+  `catalyst.vmap` offers the vectorization mapping backed by `catalyst.for_loop`.
+  In terms of feature parity with `jax.vmap`, `catalyst.vmap` supports mapping of
+  any arbitrary PyTrees as input parameters or return values. Besides, you can use:
+  - `in_axes` to specify the value(s) over which input array axes to map.
+  - `out_axes` to specify where te mapped axis should appear in the output.
+  - `axis_size` to optionally indicate the size of the axis to be mapped.
```

Comment:
If `axis_size` is provided:
-  `axis_size > batch_size` : raise `ValueError`
- `axis_size <= batch_size` : call `for_loop(0, axis_size, 1)`

I will update docstring providing more info about how it can be used.

Reply:
Sounds good, thanks! Very strange behaviour chosen by JAX 🤔